src/Microsoft.SqlTools.ServiceLayer/SchemaCompare/Contracts/SchemaCompareOptions.cs

```diff
@@ -0,0 +1,207 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.SqlServer.Dac;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Text;
+
+namespace Microsoft.SqlTools.ServiceLayer.SchemaCompare.Contracts
+{
+    /// <summary>
+    /// Class to define schema compare and publish options. 
+    /// Keeping the order and defaults same as DacFx
+    /// The default values here should also match the default values in ADS UX
+    /// </summary>
+    public class SchemaCompareOptions
```

Comment:
Could we make the default constructor initialize the defaults using reflection from the DacDeployOptions? Currently, we are already in a situation where defaults are different between the API, sqlpackage, and SSMS, and I don't want to make the situation get worse. I think the API should be the source of truth.

Reply:
Yes that sounds good.. will try